core/drivers/zynqmp_csudma.c

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2021 Foundries.io Ltd.
+ * Jorge Ramirez-Ortiz <jorge@foundries.io>
+ */
+
+#include <config.h>
+#include <io.h>
+#include <kernel/cache_helpers.h>
+#include <kernel/delay.h>
+#include <mm/core_memprot.h>
+
+#include "drivers/zynqmp_csudma.h"
+
+#define IS_ALIGNED(x, a) (((x) & ((typeof(x))(a) - 1)) == 0)
```

Comment:
`IS_ALIGNED()` is already defined in some other .c file too. Perhaps we can have a common implementation in `lib/libutils/ext/include/util.h` instead?